Magento_Theme/web/js/objects/custom-object-two.js

```diff
@@ -0,0 +1,29 @@
+define(['jquery'], function ($) {
+  return {
+    BASE_URL: 'https://app.exampleproject.test/rest/all/V1',
```

Comment:
base url is already available in the `window` object if you need that. but anyway, use `mage/url` only